### ISSUE TITLE
Benchmark report perf summary: go to zero

### ISSUE THREAD

**Issue by ge0ffrey** [TimefoldAI/timefold-solver#328]
![image](https://github.com/TimefoldAI/timefold-solver/assets/176880/04e8e7b6-85ec-4b41-a052-bc3f51280322)
the Y axis should go to zero

**Comment by triceo** [comment]
I experimented with that when I was creating this report.
In this case, it looks like a good idea.
In many other cases, "go to zero" creates an empty chart with a flat line at the top, because the difference is very small and/or very far away from zero.

**Comment by ge0ffrey** [comment]
Yes, I would compare with the old benchmark report - when to include 0 depends on the statistic.

**Comment by ge0ffrey** [comment]
When it's a flat line on the top, that's what I want to see: it shows the differences don't really matter.
Currently the new perf graph is misleading because it's not zero based.
